alfresco-transform-core-aio/alfresco-transform-core-aio-boot/src/main/resources/templates/transformForm.html

```diff
@@ -162,6 +126,10 @@ <h2>ImageMagick Test Transformation</h2>
 		</form>
 	</div>
 
+	<div style="display: inline-block;">
+		<a href="/log">Log entries</a>
+	</div>
+
```

Comment:
Actually might be better at top - it took looked a little odd on the right of ImageMagik ?